src/components/MatchingIndicator/MatchingIndicator.tsx

```diff
@@ -31,27 +31,29 @@ export const MatchingIndicator: React.FC<Props> = ({ percentage }) => {
     };
 
     return (
-        <svg {...block()} aria-label="Matching indicator">
-            <circle
-                r="9"
-                cx="50%"
-                cy="50%"
-                stroke="var(--color-background-neutral-subtle-default)"
-                fill="none"
-                strokeWidth="3"
-                strokeLinecap="round"
-            />
-            <circle
-                r="9"
-                cx="50%"
-                cy="50%"
-                stroke={computeMatchingIndicatorPercentageColor(percentage)}
-                fill="none"
-                strokeWidth="3"
-                strokeLinecap="round"
-                strokeDasharray={`${(percentage / 100) * 56}, 56`}
-            />
-        </svg>
+        <div {...block()}>
```

Comment:
if you define the props with `extends HTMLElementAttributes` (check syntax!) and then pass `rest` to block it will reuse any `classname` passed to the props. This we normally should do in all components. Also spreading `rest` on this div will ensure that other attributes (e.g. style, or data-x) will not get lost. 
Also using this approach you can save the extra `div` you use for adding a class to the wrapper, and can pass `elem` to the indicator directly.